src/main/java/org/openpnp/util/Utils2D.java

```diff
@@ -77,43 +77,33 @@ public static Point scalePoint(Point point, double scaleX, double scaleY) {
 		return new Point(point.getX() * scaleX, point.getY() * scaleY);
 	}
 	
-	public static Location calculateBoardPlacementLocation(Location boardLocation, Side boardSide, Location placementLocation) {
+
+	public static Location calculateBoardPlacementLocation(Location boardLocation, double bottom, Location placementLocation) {
```

Comment:
I don't want Side removed. Just pass in the dimensions Location in addition to Side or add a new function that just takes the entire BoardLocation.